### DISCUSSION TITLE
Welcome to Mbed CE Discussions!

### DISCUSSION THREAD

**Discussion by multiplemonomials** [mbed-ce/mbed-os#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect Mbed CE users and developers. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas and discuss pull requests / bugfixes
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ladislas** [comment]
Way to go @multiplemonomials! 👍 :)

**Comment by JojoS62** [comment]
Great, thanks 👍
